shell/platform/darwin/ios/framework/Source/FlutterKeyboardManagerTest.mm

```diff
@@ -6,9 +6,7 @@
 #import <OCMock/OCMock.h>
```

Comment:
I was hitting a crash deep in the bowels of OCMock.  Instead of trying to figure out what was wrong with the `mockPrimaryResponder` invocations, I decided to rewrite the whole thing to use `XCTestExpectation` instead of run loops and invocation callbacks.

I can put this in its own PR if that makes it easier to review.

Reply:
Scratch that, it was a simple case of needing a `__unsafe_unretained` on a `-[NSInvocation getArgument::]`.  I wound up digging myself into a hole when I tried to simplify the runloop story in this test since the callbacks to this API really want to be scheduled on a runloop, but assume the caller is doing that.  So I went back to the old code and just fixed the `__unsafe_unretained` case.